src/OptionsRESTController.php

```diff
@@ -116,7 +131,7 @@ public function register() {
 				[
 					'methods'             => WP_REST_Server::EDITABLE,
 					'callback'            => [ $this, 'update_items' ],
-					'args'                => $this->get_endpoint_args_for_item_schema( WP_REST_Server::EDITABLE ),
+					'args'                => [],
```

Comment:
@johnwatkins0 Why was this change made? It broke `OptionsRESTControllerTest::test_update_items()` and prevented the `enum` constraints and other schema requirements from being enforced from the `POST` request. 

I'm reverting for now.

Reply:
I see it was needed for WP 5.0 compat in 871bd100c040f5bbc63456a75c5e101a00434187.

I'll conditionally make this change then if the WP version is less than 5.1

Reply:
Ah, right, I meant to revisit that with a `method_exists` check or similar.

Reply:
Actually, I see no REST API error when this method is present in 5.0. Was it for 4.9-compat?

Reply:
It came up when I was in 5.0, but I see `@since 4.7` on that method, so I must have been mistaken about whatever error I was seeing.